tests/test_fiscal.py

```diff
@@ -28,24 +17,25 @@ def test_compute_hash_returns_correct_hash():
 
 
 # noinspection PyShadowingNames
-def test_validate_bad_package_with_raise_error_false(invalid_fiscal_package):
-    report = invalid_fiscal_package.validate(
+def test_validate_bad_package_with_raise_error_false():
+    report = invalid_fiscal_package().validate(
         raise_on_error=False,
         schema_only=True
     )
     assert isinstance(report, list)
-    assert 'is a required property' in report[0]
+    assert len(report) > 0
+    # assert 'is a required property' in report[0]
```

Comment:
Was it supposed to be commented out? If so, I'd say remove it

Reply:
This was testing something in the first item of the report, but since then the contents of the returned report has changed. I'm remove it.